docs/csharp/programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type.md

```diff
@@ -61,5 +63,6 @@ When you define a class or struct, you decide whether it makes sense to create a
   
 ## See also
 
-- [Equality comparisons](equality-comparisons.md)
-- [C# Programming Guide](../index.md)
+-[Equality comparisons](equality-comparisons.md)
+-[C# Programming Guide](../index.md)
+ 
```

Comment:
nit: Remove extra space

```suggestion

```

Reply:
@Youssef1313 the way it was seems to have been triggering the linter failure as the lint now fails. I'm not sure I understand what's wrong there.